docs/releases/v8_6.md

```diff
@@ -0,0 +1,55 @@
+
+### Improvements
+
+- **DietPi-Software** | [**HAProxy**](../../software/advanced_networking/#haproxy) :octicons-arrow-right-16: It received some updates and enhancements, using v2.6.0 sources now, enabled support for Prometheus metrics, updated cipher suites, and the initial password for the admin panel is now the global software password instead of hardcoded "dietpi". All config file changes affect fresh installs only, while reinstalls will never touch the config file, but update the HAProxy binary only:
+
+    ```sh
+    dietpi-software reinstall 98
+    ```
+
+- **DietPi-Software** | [**Amiberry**](../../software/gaming/#amiberry) :octicons-arrow-right-16: Our package has been updated to latest v5.3. Run
+
+    ```sh
+    dietpi-software reinstall 108
+    ```
+
+    to update your instance: <https://github.com/BlitterStudio/amiberry/releases/tag/v5.3>
+
+### Bug Fixes
+
+- **Hardware** | [**Odroid N2**](../../hardware/#odroid) :octicons-arrow-right-16: Resolved an issue where our image didn't boot from Hardkernel eMMC modules with orange PCB. Many thanks to @greentea1 for reporting this issue: <https://dietpi.com/forum/t/problem-for-my-n2/13426>
+- **DietPi-Software** | [**DietPi-VPN**](../../dietpi_tools/#dietpi-vpn) :octicons-arrow-right-16: Resolved an issue where NordVPN, IPVanish and PIA connections failed if the killswitch was enabled. Many thanks to @jonyskids for reporting this issue: <https://dietpi.com/forum/t/dietpi-vpn-blocking-443/13348>
+- **DietPi-Software** | [**DietPi-LetsEncrypt**](../../software/system_security/#lets-encrypt) :octicons-arrow-right-16: Resolved an issue where Lighttpd needed to be reloaded manually after certificate renewals to serve the new certificate. Many thanks to @Rednox for reporting this issue: <https://dietpi.com/forum/t/letsencrypt-renew-cron-issue/13471/6>
+- **DietPi-Software** | [**Radarr**](../../software/bittorrent/#radarr) :octicons-arrow-right-16: Resolved an issue where the installation on ARMv6 Raspberry Pi models (Raspberry Pi 1 and Zero (1)) failed, since Radarr v4 does not support Mono anymore. The latest v3 will be installed now on these models. Many thanks to @eddiermar for reporting this issue: <https://github.com/MichaIng/DietPi/issues/5537>
+- **DietPi-Software** | [**Lidarr**](../../software/bittorrent/#lidarr) :octicons-arrow-right-16: Precautionary, on ARMv6 Raspberry Pi models, the latest v0.8 will be installed from now on, since the upcoming v1 won't support Mono anymore.
+- **DietPi-Software** | [**Jellyfin**](../../software/media/#jellyfin) :octicons-arrow-right-16: Resolved an issue where the installation failed due to a missing config file. Many thanks to [`@TomEighty15`](https://dietpi.com/forum/u/tomeighty15/summary) for reporting this issue: <https://dietpi.com/forum/t/jellyfin-installation-error/13402>
```

Comment:
```suggestion
- **Hardware** | [**Odroid N2**](../../hardware/#odroid) :octicons-arrow-right-16: Resolved an issue where our image didn't boot from Hardkernel eMMC modules with orange PCB. Many thanks to [@greentea1](https://dietpi.com/forum/u/greentea1/summary){: class="nospellcheck"} for reporting this issue: <https://dietpi.com/forum/t/problem-for-my-n2/13426>
- **DietPi-Software** | [**DietPi-VPN**](../../dietpi_tools/#dietpi-vpn) :octicons-arrow-right-16: Resolved an issue where NordVPN, IPVanish and PIA connections failed if the killswitch was enabled. Many thanks to [@jonyskids](https://dietpi.com/forum/u/jonyskids/summary){: class="nospellcheck"} for reporting this issue: <https://dietpi.com/forum/t/dietpi-vpn-blocking-443/13348>
- **DietPi-Software** | [**DietPi-LetsEncrypt**](../../software/system_security/#lets-encrypt) :octicons-arrow-right-16: Resolved an issue where Lighttpd needed to be reloaded manually after certificate renewals to serve the new certificate. Many thanks to [@Rednox](https://dietpi.com/forum/u/rednox/summary){: class="nospellcheck"} for reporting this issue: <https://dietpi.com/forum/t/letsencrypt-renew-cron-issue/13471/6>
- **DietPi-Software** | [**Radarr**](../../software/bittorrent/#radarr) :octicons-arrow-right-16: Resolved an issue where the installation on ARMv6 Raspberry Pi models (Raspberry Pi 1 and Zero (1)) failed, since Radarr v4 does not support Mono anymore. The latest v3 will be installed now on these models. Many thanks to @eddiermar for reporting this issue: <https://github.com/MichaIng/DietPi/issues/5537>
- **DietPi-Software** | [**Lidarr**](../../software/bittorrent/#lidarr) :octicons-arrow-right-16: Precautionary, on ARMv6 Raspberry Pi models, the latest v0.8 will be installed from now on, since the upcoming v1 won't support Mono anymore.
- **DietPi-Software** | [**Jellyfin**](../../software/media/#jellyfin) :octicons-arrow-right-16: Resolved an issue where the installation failed due to a missing config file. Many thanks to [@TomEighty15](https://dietpi.com/forum/u/tomeighty15/summary){: class="nospellcheck"} for reporting this issue: <https://dietpi.com/forum/t/jellyfin-installation-error/13402>
```

Reply:
Ok.